lib/cli/src/initiate.ts

```diff
@@ -307,5 +307,11 @@ export default function (options: CommandOptions, pkg: IPackage): Promise<void>
   }
   done();
 
-  return installStorybook(projectType, options);
+  const cleanOptions = { ...options };
+  if (options.storyFormat === StoryFormat.MDX) {
+    logger.warn('   Using mdx format is deprecated. JS or TS format already embeds mdx stories !');
```

Comment:
```suggestion
    logger.warn('   The MDX CLI template is deprecated. The JS and TS templates already embed mdx stories!');
```

Reply:
This is misleading. MDX is not deprecated, just the template that generates it exclusively.